.github/workflows/python-test.yaml

```diff
@@ -3,8 +3,12 @@ name: Python check
 on:
   push:
     branches: [ main ]
+    paths:
+      - "**.py"
   pull_request:
     branches: [ main ]
+    paths:
+      - "**.py"
```

Comment:
This workflow also does E2E tests, so it could be affected by frontend changes as well. Is it possible to say "run this if the change affects anything but MD files"? I think quite a few changes could affect this workflow.

Reply:
Yep we can